File: pingyougou-manager-web/src/main/webapp/js/gngularBran/specificationService.js
 app.service('specificationService',function($http){
  
    	this.fanAll=function(pageTole,pageSize,spnate){
    		return $http.post('../specification/selectList.do?pageTole='+pageTole+'&pageSize='+pageSize,spnate);
    	}
    	
    	this.updateInsert=function(id){
    		return $http.get('../specification/updateInsert.do?id='+id);
    	}
    	
    	this.deleteSp=function(deleteSpIds){
    		alert(deleteSpIds);
    	return $http.get('../specification/deleteSp.do?ids='+deleteSpIds);
    	}
    	
    	this.insertList=function(specificationPojo){
    	return $http.post('../specification/insertList.do',specificationPojo);
    	}
    	
    	this.updateSp=function(specificationPojo){
    	return $http.post('../specification/updateSp.do',specificationPojo);
    	}
    	
    })

File: pingyougo-sellergoods-interface/src/main/java/cn/pingyougou/sellergoodsServiceInterface/SpecificationService.java
package cn.pingyougou.sellergoodsServiceInterface;
import PingYouGouPojo.SpecificationPojo;
import cn.pinyougou.pojo.TbSpecification;
import entryPingYouGou.PageToel;

public interface SpecificationService {

	PageToel<TbSpecification> selectList(int pageTole,int pageSize,String pageStr);

	int insertList(SpecificationPojo specificationPojo);

	SpecificationPojo updateInsert(long id);

	void updateSp(SpecificationPojo specificationPojo);
 
	int deleteSp(long[] ids);

}
